src/_locales/zh_TW/messages.json

```diff
@@ -0,0 +1,250 @@
+    "message": "某些設置由系統管理員管理，無法更改。",
+    "description": "The message, which appears, when settings are pre-defined (as managed options) by administrators.",
+    "hash": "148604ebbeb7df28974d8233f1418d1f"
+  },
+  "optionIsDisabledBecauseManaged": {
+    "message": "此選項已禁用，因為它已由系統管理員配置。",
+    "description": "The title (tooltip) shown, when hovering over a disabled, managed option.",
+    "hash": "50fb70682c64e1a1fb4d8884b33cca64"
+  },
+  "optionLearnMore": {
+    "message": "了解更多",
+    "description": "When a link to an explainer needs to be added, this is the link text.",
+    "hash": "9085b09f4748be12ab0c241d2e643b31"
+  },
+  "optionsResetButton": {
+    "message": "將所有設定重置為預設值",
+    "description": "The button to delete all current settings and load the defaults, shown in the add-on settings.",
+    "hash": "4bb60c6c7e7b9e2ff987c244e76ee044"
+  },
+  "optionMastodonHandle": {
+    "message": "您的 Mastodon 帳號：",
+    "description": "This is an option shown in the add-on settings. You can enter your handle in the format mastodon@server.com there.",
+    "hash": "c3056895503de22349f1b77062b3bbd3"
+  },
+  "optionMastodonHandlePlaceholder": {
+    "message": "you@mastodon.example",
+    "description": "This is placeholder shown for the Mastodon handle input. Try to give an example on format the user has to use to enter the ID as  mastodon@server.com here. Do not use a real server name and do not promote any really existing user.",
+    "hash": "9a20059d2d16dfb7e731226187c04836"
+  },
+  "optionMastodonHandleHelper": {
+    "message": "請以 “username@serverdomain” 格式輸入您的 Mastodon 帳號。",
+    "description": "An explanatory text shown to the user in the settings to explain what format they have to enter the Mastodon handle. It should not hint to any example (that's what the placeholder is for), but explain the general format.",
+    "hash": "5e2a6e34c5fed46740dc238f1cc7768a"
+  },
+  "optionRedirectInMainWindow": {
+    "message": "防止彈出視窗。",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "dc056fc373c7161104ad81b28884a017"
+  },
+  "optionRedirectInMainWindowDescr": {
+    "message": "以重導向主頁上的操作取代彈出視窗。",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "ba0f60f1ae0c6d6785d5eb410247eb11"
+  },
+  "translatorCredit": {
+    "message": "此插件已由 $TRANSLATORS$ 翻譯成英文。",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/rugk",
+    "description": "The link to the translator's GitHub profile.",
+    "hash": "4775fb2f88fd33d02b871e5021dec3e9"
+  },
+  "translatorUsername": {
+    "message": "rugk",
```

Comment:
Please pass your own name/reference here. (same for the link above)
See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translations for how these placeholders are translated, especially [the official doc](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales).

Reply:
I already change that at https://github.com/ryanho/mastodon-simplified-federation/commit/b8390e7f805b92d610904249c6ddd1cdb5c70187, but I don't know how to add it to this PR.

Reply:
Just push the commit/change to the same branch that your current PR uses (i.e. `master`).

See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches for more information.

Reply:
Yes. These changes already commit to same branch and push to Github.

Reply:
Okay, then it is indeed crazy why they don't appear here?

As I can see the commit at the link you linked above _and_ it also states there that it is in the `master` branch, I really have no clue what is wrong here. I can [also see the changes in the diff view](https://github.com/rugk/mastodon-simplified-federation/compare/master...ryanho:mastodon-simplified-federation:master), whcih points me to this PR – which does not contain the changes(?).

Really, that is not how it is supposed to work…
Thus do you maybe wanna [contact GitHub support](https://support.github.com/features/pull-requests) (button at the bottom) to let them debug that?

Reply:
Seems like free account can't create a support ticket. I have to post my question on  GitHub Community and wait for answer. Or maybe just close this PR and try again?

Reply:
OK. Further commits finally show in this PR.

Reply:
Ah oh okay, I know this problem and here is what may have caused this:

Please do also note one _important_ precautionary warning of me: In case you did not notice, you have committed (i.e. created commits) with potentially sensitive data like your (full/private) name or mail.
Anyone, who clones your fork/this repo can view these private details, exactly as you've committed them. Also, if this PR gets merged, this will stay in the (commit) history of this project and will thus hard to be removed later.

So if this was unintentional and you want that data to be changed/removed first, better do it now. :smiley: [Here is a good resource](https://stackoverflow.com/a/750182/5008962) ([actually from GitHub's help](https://help.github.com/en/articles/changing-author-info)) on how to change your name/mail afterwards.
Also note that when setting your git [username](https://help.github.com/en/articles/setting-your-username-in-git) and [especially your email](https://help.github.com/en/articles/setting-your-commit-email-address), you can actually write/choose anything there. As for GitHub, it is recommend to use your GitHub username and – if you want to keep your mail private – the GitHub provided `noreply` address. You can [even block pushes of commits with private mails](https://help.github.com/en/articles/blocking-command-line-pushes-that-expose-your-personal-email-address).

So far just the possibilities. I just wanted to serve this as a friendly reminder, so you don't notice this only when it's too late. What you'll do depends solely on you. :smile:

Reply:
BTW, do you have a link to the _GitHub Community_ post you created? I would very like to follow that.

Reply:
> BTW, do you have a link to the _GitHub Community_ post you created? I would very like to follow that.

Nothing interesting there, I think.
https://github.com/orgs/community/discussions/38743